app/src/main/java/com/example/football_manager/News/EditNews.kt

```diff
@@ -0,0 +1,97 @@
+package com.example.football_manager.News
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+
+
+@Composable
+fun EditNews(id: Int, navController: NavHostController){
+
+    val errors = remember { mutableStateListOf<String>() }
+    val EDIT_NEWS_TITLE_MIN_LENGTH = 3
+    val EDIT_NEWS_TITLE_MAX_LENGTH = 30
+    val EDIT_NEWS_CONTENT_MIN_LENGTH = 10
+    val EDIT_NEWS_CONTENT_MAX_LENGTH = 120
```

Comment:
You use these constants both here and in the create news view. 
Maybe move them somewhere else so you can reuse.

Reply:
I have specific variables related to the EditNews.kt and to CreateNews.kt. I have done the same for activities and the laboratory assignment. Would Peter react in a bad way or give us less in grade? 
The only reason I ask is to improve my programming skills.

Reply:
Tänk DRY. 
Nästan den skulle vara bra att ha en egen fil med alla dessa, oberoende på vilken view.

Reply:
Kör globala variabler